### ISSUE TITLE
Issue with the snap interaction & how to automatically select/modify a feature on drawend?

### ISSUE THREAD

**Issue by akc4** [MelihAltintas/vue3-openlayers#9]
1. In the following example, when drawing, the `snap` interaction works only for the first drawn polygon. Or if you first select and modify one of the two polygons and then start drawing a polygon, the snap interaction doesn't work.
-However the snap interactions works on the `modify interaction`.

2. I am having a hard time figuring out how to automatically select / modify a polygon when I finish drawing it on the `drawend` event.
Adding the following to the `drawend` function doesn't quite work and I don't think I am doing it the right way.
```
selectedFeatures.value = event.feature
modifyEnabled.value = true
```

https://codesandbox.io/s/vue3-openlayers-forked-qqrqx?file=/src/App.vue

**Comment by MelihAltintas** [comment]
1) i did not reproduce the issue
2)
you should add this line to drawend function
`selectedFeatures.value.push(event.feature)`
and bind selectedFeatures to features props of select

example with 0.1.28:  https://codesandbox.io/s/vue3-openlayers-forked-lbvtf?file=/src/App.vue

**Comment by MelihAltintas** [comment]
i fixed this error with 0.1.29 :)
